src/OnlineRetailPortal.Web/Models/Category.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Web
+{
+    public enum Category
+    {
```

Comment:
What should be the default category?
Should it be properties or something generic?

Reply:
it is mandatory field